pytrip/vdx.py

```diff
@@ -933,9 +962,7 @@ def add_dicom_contour(self, dcm):
         offset.append(float(self.cube.yoffset))
         offset.append(float(min(self.cube.slice_pos)))
         self.contour.append(
-            Contour(pytrip.res.point.array_to_point_array(
-                np.array(
-                    dcm.ContourData, dtype=float), offset)))
+            Contour(pytrip.res.point.array_to_point_array(np.array(dcm.ContourData, dtype=float), offset)))
```

Comment:
if you have figured out what array_to_point_array() is doing, can you update the docstring?